tools/wheelmaker.py

```diff
@@ -124,6 +124,15 @@ def arcname_from(name):
                 size += len(block)
         self._add_to_record(arcname, self._serialize_digest(hash), size)
 
+    def add_top_level_txt(self, packages):
+        """Write top_level.txt file for the distribution"""
```

Comment:
is there any canonical doc about wheels or eggs that specified what the top_level.txt file is? would be good to link there